### PR TITLE
add: Input Component

### PR DESCRIPTION
**Pull request by shilpe26**
Made Input Component with features:-
🔹Simple Input field
🔹Input field with Error Validation

### CODE REVIEW

File: components/input.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (Mohasin-Haque) [inline components/input.html:7]: You can add more meta informations to give it a personalise touch.. like
<meta name="author" content"your name">
<meta name="description" content="about your ui">

File: components/input.html [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    <!-- css style links -->
+
+    <link rel="stylesheet" href="../css/component_styles/input_component.css">
+
+    <!-- title -->
+    <title>Image</title>
+
+    <!-- Font Awesome link -->
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"
+        integrity="sha384-Bfad6CLCknfcloXFOyFnlgtENryhrpZCe29RTifKEixXQZ38WheV+i/6YWSzkz3V" crossorigin="anonymous" />
```
Comment (Mohasin-Haque) [inline components/input.html:19]: you can include this in main css file so that you need not to repeatadly use it

File: components/input.html [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+                        sandbox="allow-scripts allow-same-origin">
+                    </iframe>
+                </div>
+                <!-- Footer -->
+                <footer>
+                    <div class="prev">
+                        <i class="fas fa-angle-left"></i>
+                        <a href="./image.html">Image</a></div>
+                    <!-- <div class="next"><a href="">Alert</a>
+                        <i class="fas fa-angle-right"></i>
```
Comment (Mohasin-Haque) [inline components/input.html:132]: Remove the commented html

File: css/component_styles/input.css [hunk 4]
```diff
@@ -0,0 +1,54 @@
+.input-group {
+    color: #333;
+    font-size: .8rem;
+    line-height: 20px;
+    margin: 0 20px 10px;
```
Comment (Mohasin-Haque) [inline css/component_styles/input.css]: if possible use relative units

**Review by Mohasin-Haque — COMMENTED** [review]
Nicely done!! So neat and tidy code.. Please work on suggestions if you can!  Rest is all good. you can go ahead then.

File: components/input.html (hunk 3 above)
Comment (shilpe26) [reply components/input.html:132]: This I put for the next page when I add.

File: components/input.html (hunk 2 above)
Comment (shilpe26) [reply components/input.html:19]: Yes, I will do it after 2 more component together

**Comment by shilpe26** [comment]
Done all the changes needed.👍🏻

**Pull request merged**